decs.d.ts

```diff
@@ -14,5 +16,32 @@ declare module '@gridsuite/commons-ui' {
         snackInfo: (snackInputs: SnackInputs) => void;
     }
 
+    type ParameterType = {
+        type: string;
+        name: string;
+        possibleValues: string[];
+        defaultValue: string;
+    };
+
+    interface IFlatParameters {
+        paramsAsArray: ParameterType[];
+        initValues: Record<string, string | boolean>;
+        onChange: (
+            paramName: string,
+            newValue: string,
+            isInEdition: boolean
+        ) => void;
+        variant?: 'outlined' | 'standard' | 'filled';
+        showSeparator?: boolean;
+    }
+
     export function useSnackMessage(): UseSnackMessageReturn;
+
+    export const FlatParameters: FunctionComponent<IFlatParameters>;
+
+    export function useDebounce(
+        [string]: () => void,
+        [string]: number,
+        ...args: any[]
```

Comment:
Why this line ?

Reply:
one day if we want to add some props for useDebounce .. 
but I will delete it 
thank you